proto/vtctldata.proto

```diff
@@ -260,6 +260,8 @@ message Workflow {
     repeated string tags = 15;
     int64 rows_copied = 16;
     ThrottlerStatus throttler_status = 17;
+    string tablet_types = 18;
+    string cells = 19;
```

Comment:
Can we make these `repeated string` so that it's a slice for each? We're doing that for all other fields in this newer message. We can then split the stored CSV value in the DB.

Reply:
+1 also we could make tablet_types a `repeated topodata.TabletType`